tests/components/updater/test_init.py

```diff
@@ -157,6 +157,31 @@ def mock_get_uuid_fixture():
         await updater.get_newest_version(hass, MOCK_HUUID, False)
 
 
+async def test_fallback_to_get_if_post_fails(hass, aioclient_mock):
+    """Test we handle response error while fetching new version."""
+    aioclient_mock.post(updater.UPDATER_URL, status=500)
+    aioclient_mock.get(updater.UPDATER_URL, json=MOCK_RESPONSE)
+    with patch(
+        "homeassistant.helpers.system_info.async_get_system_info",
+        Mock(return_value=mock_coro({"fake": "bla"})),
```

Comment:
Please import `patch` from asynctest so we don't need to use `mock_coro`.
```suggestion
        return_value={"fake": "bla"},
```

Reply:
I fixed them all